commons/src/test/java/org/sonarsource/analyzer/commons/EntropyDetectorTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * SonarSource Analyzers Commons
+ * Copyright (C) 2009-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.analyzer.commons;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class EntropyDetectorTest {
+
+  @ParameterizedTest
+  @MethodSource("stringPerLevelForEntropyScore")
+  void test_progressive_entropy_score_sensibility(String input, int currentSensibility) {
```

Comment:
We should use camelCase for test method names I believe

Reply:
There is no such convention in this project, mostly, snake_case is used. I will change to camelCase as it is preferred by me and to be consistent with `LatinAlphabetDetectorTest`.